tools/localization-web/src/styles.css

```
@@ -0,0 +1,271 @@
+  padding: 0.4rem 0.6rem;
+  text-align: left;
+  vertical-align: top;
+}
+
+td.text {
+  max-width: 24rem;
+}
+
+td.source {
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.14.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)